dockstore-webservice/src/main/java/io/dockstore/webservice/resources/LambdaEventResource.java

```diff
@@ -72,6 +72,34 @@ public List<LambdaEvent> getLambdaEventsByOrganization(@ApiParam(hidden = true)
         return lambdaEventDAO.findByOrganization(organization, offset, limit, authorizedRepos);
     }
 
+    @GET
+    @Timed
+    @UnitOfWork(readOnly = true)
+    @Path("/{userid}/{organization}")
+    @Operation(operationId = "getUserLambdaEventsByOrganization", description = "Get all of the Lambda Events for the given user and GitHub organization.",
+            security = @SecurityRequirement(name = ResourceConstants.JWT_SECURITY_DEFINITION_NAME))
+    @ApiOperation(value = "See OpenApi for details")
+    public List<LambdaEvent> getUserLambdaEventsByOrganization(@ApiParam(hidden = true) @Parameter(hidden = true, name = "user")@Auth User authUser,
+           @ApiParam(value = "userid", required = true) @PathParam("userid") long userid,
+           @ApiParam(value = "organization", required = true) @PathParam("organization") String organization,
+           @ApiParam(value = PAGINATION_OFFSET_TEXT) @QueryParam("offset") @DefaultValue("0") String offset,
```

Comment:
Anyone recall why some endpoints use String offset while others use integer? https://github.com/dockstore/dockstore/blob/2f677b50e2f7a55cf9b1a818a0137531411c7424/dockstore-webservice/src/main/java/io/dockstore/webservice/resources/EventResource.java#L101
Integer seems more intuitive to me, but String seems to occur often enough that I wonder if there is a reason for it.